### PR TITLE
[12.0][MIG] account_move_budget

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Standard migration of module account_move_budget to v12

cc ~ @Eficent @jbeficent

### CODE REVIEW

File: account_move_budget/models/account_move_budget.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    description = fields.Char(
+        track_visibility='onchange',
+    )
+    date_range_id = fields.Many2one(
+        comodel_name='date.range',
+        string='Date range',
+    )
+    date_from = fields.Date(
+        required=True,
+        string='From',
```
Comment (nikul-serpentcs) [inline account_move_budget/models/account_move_budget.py]:
```suggestion
        string='From Date',
```

File: account_move_budget/models/account_move_budget.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        track_visibility='onchange',
+    )
+    date_to = fields.Date(
+        required=True,
+        string='To',
```
Comment (nikul-serpentcs) [inline account_move_budget/models/account_move_budget.py]:
```suggestion
        string='To Date',
```

**Review by fclementic2c — APPROVED** [review]
Functional tests ok

**Review by HviorForgeFlow — APPROVED** [review]

**Comment by fclementic2c** [comment]
can we merge this PR ?

**Comment by HviorForgeFlow** [comment]
@fclementic2c sure

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:  LGTM

**Pull request merged**